### ISSUE TITLE
run_test_file() does not see global variables?

### ISSUE THREAD

**Issue by KKPMW** [markvanderloo/tinytest#7]
Maybe I am doing something wrong, but the following example produces an error:

file **testfail.R**:

```
x <- rnorm(10)
g <- rep(c("a", "b"), 5)
expect_warning(tapply(x, g, mean))
```

In R:

```
library(tinytest)
run_test_file("testfail.R")

Error in tapply(x, g, mean) : object 'g' not found
```

First time trying to use the library, so maybe I am missing something obvious?

```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin18.2.0 (64-bit)
Running under: macOS  10.14.5

Matrix products: default
BLAS: /opt/local/Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.dylib
LAPACK: /opt/local/Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] tinytest_0.9.3 colorout_1.2-0 setwidth_1.0-4

loaded via a namespace (and not attached):
[1] compiler_3.5.1 tools_3.5.1
```

**Comment by markvanderloo** [comment]
Hi there, and thanks for your interest in the package.

You are not doing anything wrong: it's actually issue #6 and it has been solved in the the development version of 0.9.4. (take a look there to see how to test it)

I'm currently chasing a bug that was submitted to me by dm. Once that is flushed out I'll publish new version.

**Comment by KKPMW** [comment]
Ah, I see. The github version is actually 0.9.4, not 0.9.3. Missed that. Thanks for a fast reply!
